polkadot/node/subsystem-bench/src/lib/network.rs

```diff
@@ -149,6 +149,9 @@ impl RateLimit {
 
 /// A wrapper for both gossip and request/response protocols along with the destination
 /// peer(`AuthorityDiscoveryId``).
+// Dev note: clippy warning is because RequestFromNode is at least 1160 bytes and second
+// largest variant is MessageFromPeer which is at least 240 bytes.
+#[allow(clippy::large_enum_variant)]
```

Comment:
Can you not just box them instead of adding a clippy allow that will never be fixed.

Reply:
Yeah that's fair; I just wanted to keep changes to a minimum and not risk more churn, but lemme have a go at this and I geuss the sc_service::Error as well